src/download-cid.js

```diff
@@ -0,0 +1,144 @@
+const { join } = require('path')
+const fs = require('fs-extra')
+const i18n = require('i18next')
+const isIPFS = require('is-ipfs')
+const { clipboard, app, shell } = require('electron')
+const logger = require('./common/logger')
+const { IS_MAC } = require('./common/consts')
+const setupGlobalShortcut = require('./setup-global-shortcut')
+const { selectDirectory } = require('./dialogs')
+const dock = require('./dock')
+const showPrompt = require('./prompt')
+const { showDialog } = require('./dialogs')
+
+const CONFIG_KEY = 'downloadHashShortcut'
+
+const SHORTCUT = IS_MAC
+  ? 'Command+Control+H'
+  : 'CommandOrControl+Alt+D'
+
+async function saveFile (dir, file) {
+  const location = join(dir, file.path)
+  await fs.outputFile(location, file.content)
+}
+
+async function getCID () {
+  const text = clipboard.readText().trim()
+
+  const { button, input } = await showPrompt({
+    title: i18n.t('downloadCidContentDialog.title'),
+    message: i18n.t('downloadCidContentDialog.message'),
+    defaultValue: isIPFS.cid(text) ? text : '',
+    buttons: [
+      i18n.t('downloadCidContentDialog.action'),
+      i18n.t('cancel')
+    ],
+    window: {
+      width: 460,
```

Comment:
Should we make it a tiny bit wider so the entire base32 cid fits? 
On my Linux box it gets cut off, but may be to custom font settings I have:

> ![2020-04-22-002352_460x120_scrot](https://user-images.githubusercontent.com/157609/79920518-93220b80-8430-11ea-9bdb-775d29cc9a07.png)

(if it already fits on Mac and Win and is only a Linux issue, ignore this comment)

Reply:
+1 😊

Reply:
Done!